infinispan-remote/infinispan-server-testconfig/wildfly-trimmed-config.xml

```diff
@@ -94,7 +94,7 @@
             </formatter>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:deployment-scanner:2.0">
-            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="50" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
+            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="1" auto-deploy-exploded="true" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
```

Comment:
Why do you need this change? Some explanation in the commit message or in the file would help. I assume is because you need to deploy the stored procedures